gateway-js/src/__tests__/loadSupergraphSdlFromStorage.test.ts

```diff
@@ -34,253 +41,571 @@ describe('loadSupergraphSdlFromStorage', () => {
       compositionId: null,
 
     });
+    expect(result).toMatchInlineSnapshot(snapshot);
+  });
 
-    expect(result).toMatchInlineSnapshot(`
-      Object {
-        "id": "originalId-1234",
-        "supergraphSdl": "schema
-        @core(feature: \\"https://specs.apollo.dev/core/v0.2\\")
-        @core(feature: \\"https://specs.apollo.dev/join/v0.2\\", for: EXECUTION)
-        @core(feature: \\"https://specs.apollo.dev/tag/v0.1\\")
-      {
-        query: Query
-        mutation: Mutation
-      }
-
-      directive @core(feature: String!, as: String, for: core__Purpose) repeatable on SCHEMA
+  it('Queries alternate Uplink URL if first one fails', async () => {
+    mockSupergraphSdlRequest().reply(500);
```

Comment:
I think you need the `nock.cleanAll()` because this is missing. Should be able to remove the `nock.cleanAll()` since the requests will match up correctly with this in place.
```suggestion
    mockSupergraphSdlRequest('originalId-1234').reply(500);
```

Reply:
Fixed